bin/setup

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
```

Comment:
Only personal feeling here, not sure about the majority or why bundler add them.